stage1/usr_from_kvm/lkvm.mk

```diff
@@ -5,39 +5,64 @@ LKVM_BINARY := $(LKVM_SRCDIR)/lkvm-static
 LKVM_ACI_BINARY := $(ACIROOTFSDIR)/lkvm
 LKVM_GIT := https://kernel.googlesource.com/pub/scm/linux/kernel/git/will/kvmtool
 # just last published version (for reproducible builds), not for any other reason 
-# cat be set to master to follow to upstream
+# can be set to master to follow to upstream change (please uncomment include makelib/git-refresh.mk also)
```

Comment:
The commented out part looks like a dead code, maybe remove it (if it is not desirable to have it configurable via `./configure` flag in future).